src/absil/ilsign.fs

```diff
@@ -252,7 +252,7 @@ open System.Runtime.InteropServices
 
     // Returns a CLR Format Blob public key
     let GetPublicKeyForKeyPair keyBlob =
-        use rsa = new RSACryptoServiceProvider()
+        use rsa = System.Security.Cryptography.RSA.Create()
```

Comment:
i used fullname because `RSA.Create` was not enough (the method Create not defined for ..) but the first `RSA.Create` was ok (in the same file), so dunno